src/pages/AddCar/AddCar.jsx

```diff
@@ -0,0 +1,120 @@
+import { FaPlus, FaTrash } from 'react-icons/fa';
+import React, { useEffect, useState } from 'react';
+import './index.css';
+
+const AddCar = () => {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [color, setColor] = useState('');
+  const [year, setYear] = useState('');
+  const [price, setPrice] = useState('');
+  const [type, setType] = useState('');
+  const [formInfo, setFormInfo] = useState({});
+  const [images, setImages] = useState([]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setFormInfo(
+      {
+        ...formInfo,
+        name,
+        description,
+        color,
+        year,
+        price,
+        type,
+        images,
+      },
+    );
+    setName('');
+    setDescription('');
+    setColor('');
+    setYear('');
+    setPrice('');
+    setType('');
+    setImages([]);
+  };
+
+  useEffect(() => {
+    console.log(formInfo);
+  }, [formInfo]);
+
+  const addImage = () => {
+    const newImages = [...images, ''];
+    setImages(newImages);
+  };
+
+  const deleteImage = (i) => {
+    const newImages = [...images];
+    newImages.splice(i, 1);
+    setImages(newImages);
+  };
+
+  const handleImage = (e, index) => {
+    const newImages = [...images];
+    newImages[index] = e.target.value;
+    setImages(newImages);
+  };
+
+  return (
+    <section id="add_car">
+      <div className="form__container">
+        <h2>Add A Car</h2>
+        <form onSubmit={handleSubmit}>
+          <label htmlFor="name">
+            Name:
+            <input type="text" required onChange={(e) => setName(e.target.value)} id="name" value={name} />
+          </label>
+          <label htmlFor="description">
+            Description:
+            <textarea onChange={(e) => setDescription(e.target.value)} value={description} id="description" />
+          </label>
+          <label htmlFor="color">
+            Color
+            <input type="text" onChange={(e) => setColor(e.target.value)} value={color} id="color" />
+          </label>
+          <label htmlFor="year">
+            Year
+            <input type="text" required onChange={(e) => setYear(e.target.value)} value={year} id="year" />
+          </label>
+          <label htmlFor="price">
+            Price
+            <input type="text" required onChange={(e) => setPrice(e.target.value)} value={price} id="price" />
```

Comment:
I suggest changing the input type to the number or if exists decimal. Writing text in this input field should be disabled. Only numbers should be accepted. Apart from our model validation, it would be great if we implemented input validations too